docs/servo_power_monitor/src/index.ts

```diff
@@ -79,21 +79,35 @@ selectDUTSerialButton.addEventListener('click', async () => {
 });
 
 const form = document.getElementById('form') as HTMLFormElement;
+const input = document.getElementById('input') as HTMLInputElement;
 form.addEventListener('submit', async e => {
   e.preventDefault();
 
   if (DUTPort === undefined) {
     overlay.classList.remove('closed');
-  } else {
-    const input = document.getElementById('input') as HTMLInputElement | null;
-    if (input === null) return;
-    const DUTWritable = DUTPort.writable;
-    if (DUTWritable === null) return;
-    const DUTWriter = DUTWritable.getWriter();
-    await DUTWriter.write(encoder.encode(input.value + '\n'));
-    input.value = '';
-    await DUTWriter.releaseLock();
+    return;
+  }
+  const DUTWritable = DUTPort.writable;
+  if (DUTWritable === null) return;
+  const DUTWriter = DUTWritable.getWriter();
+  await DUTWriter.write(encoder.encode(input.value + '\n'));
+  input.value = '';
+  await DUTWriter.releaseLock();
+});
+
+input.addEventListener('keydown', async e => {
+  if (DUTPort === undefined) {
+    overlay.classList.remove('closed');
+    return;
+  }
+  const DUTWritable = DUTPort.writable;
+  if (DUTWritable === null) return;
+  const DUTWriter = DUTWritable.getWriter();
+  if (e.ctrlKey && e.key === 'c') {
+    const cancel_cmd = '\x03\n';
```

Comment:
Not sure about JS style, but in my experience consts are usually defined at the top of the file. Could you move this to the top with the other const (intervalMs)